crates/nu-std/lib/help.nu

```diff
@@ -98,57 +98,55 @@ def "nu-complete list-externs" [] {
     $nu.scope.commands | where is_extern | select name usage | rename value description
 }
 
-def print-help-header [
+def build-help-header [
     text: string
     --no-newline (-n): bool
 ] {
     let header = $"(ansi green)($text)(ansi reset):"
 
     if $no_newline {
-        print -n $header
+        $header
     } else {
-        print $header
+        $header ++ "\n"
     }
 }
 
-def show-module [module: record] {
-    if not ($module.usage? | is-empty) {
-        print $module.usage
-        print ""
-    }
-
-    print-help-header -n "Module"
-    print $" ($module.name)"
-    print ""
-
-    if not ($module.commands? | is-empty) {
-        print-help-header "Exported commands"
-        print -n "    "
-
-        let commands_string = (
-            $module.commands
-            | each {|command|
-                $"($command) (char lparen)($module.name) ($command)(char rparen)"
+def build-module-page [module: record] {
+    let usage = (if not ($module.usage? | is-empty) {[
+        $module.usage
+        ""
+    ]} else [])
+
+    let name = [
+        $"(build-help-header -n "Module") ($module.name)"
+        ""
+    ]
+
+    let commands = (if not ($module.commands? | is-empty) {[
+        (build-help-header -n "Exported commands")
+        $"    (
+            $module.commands | each {|command|
+                $'($command) (char lparen)($module.name) ($command)(char rparen)'
             }
-            | str join ", "
-        )
-
-        print $commands_string
-        print ""
-    }
-
-    if not ($module.aliases? | is-empty) {
-        print-help-header "Exported aliases"
-        print $"    ($module.aliases | str join ', ')"
-        print ""
-    }
-
-    if ($module.env_block? | is-empty) {
-        print $"This module (ansi cyan)does not export(ansi reset) environment."
-    } else {
-        print $"This module (ansi cyan)exports(ansi reset) environment."
-        print (view source $module.env_block)
-    }
+            | str join ', '
+        )"
+        ""
+    ]} else [])
```

Comment:
i see you using this `if $cond {} else []` syntax, and i think it's not intuitive, (i was surprised that it even works tbh)

i think

```rs
if $cond {} else {}
````

 is better